libvcx/src/aries/handlers/connection/connection.rs

```diff
@@ -117,13 +124,13 @@ impl Connection {
         }.into()
     }
 
-    pub fn state(&self) -> u32 {
+    pub fn state(&self) -> ConnectionState {
```

Comment:
Let's rename this to `get_state`. Function names make more sense as verbs rather than nouns (as opposed to fields).

Reply:
agreed